gulpfile.js

```diff
@@ -63,6 +65,19 @@ gulp.task('dev:reload', function () {
   })
 })
 
+const replacePaths = [
```

Comment:
We can get rid of this list just by checking if an override file exists on disk or not. If it does then we replace. If not then we don't.